deepcell/utils/transform_utils_test.py

```diff
@@ -86,6 +87,7 @@ def test_pixelwise_transform_2d(self):
 
                 self.assertEqual(pw_img.shape[-1], 4)
                 self.assertEqual(pw_img_dil.shape[-1], 4)
+                assert(np.all(np.equal(pw_img[..., 0] + pw_img[..., 1] + pw_img[..., 2], img > 0)))
```

Comment:
I added in an assert statement that makes sure that all pixels which were within a labeled cell are part of interior or border class. I wasn't sure how to do this with self.assert, since we want to do elementwise assertion of the entire array

Reply:
No worries, there's nothing particularly special about `self.assert`.